regression/cbmc-incr-oneloop/unwind-forever2/test.desc

```diff
@@ -1,7 +1,11 @@
-CORE
+THOROUGH
 main.c
---incremental-check main.0
-^EXIT=142$
+--incremental-loop main.0 --unwinding-assertions
+^EXIT=(142|14)$
 ^SIGNAL=0$
 --
 ^warning: ignoring
+--
+Tests whether goto-symex does not stop early when it cannot figure out the
+loop bound and there is no error in the loop.
+Mac OS exit code is 14
```

Comment:
Why are the exit codes different? Is the exit code not under our control in this situation? What process is receiving the SIGALRM - perl, or cbmc? or something else?

Reply:
perl - the exit codes seem to be different on linux vs mac os.